Core1Platform/AdministratingCore1/02_ManageUsers.md

```diff
@@ -0,0 +1,260 @@
+    - **Generate password and send via email**  
+      A password is generated automatically, and it is sent by Actindo to the email address specified below. 
+    - **Enter password and send by email**    
+      You define an individual password, and it is sent by Actindo to the email address specified below. 
+    - **Specify and save password**   
+      You define an individual password and save it. It is not sent via email.   
+    - **Do not change password**   
+      Use this option if you want to change the user's data, but you do not want to change the password resp. the password to be sent. 
+
+4. If desired, disable the *Universal Second Factor (U2F)* toggle.
+  If the *Universal Second Factor (U2F)* is disabled, the user will no longer need to use the *Yubikey* to log in. 
+
+5. If desired, edit the user's personal data.
+
+6. Click the [SAVE] button.   
+
+7. If desired, change the group assignment of the user. For detailed information, see [Assign user to user group](#assign-user-to-user-group "Assign user to user group").
+
+
+
+## Remove user from a group
+
+When editing a user, you can remove a user from a group to which he or she is assigned.  
+Alternatively, you can remove a user from a group during editing the groups, see [Remove users from a group via the group management](../AdministratingCore1/03_ManageGroups.md#remove-users-from-a-group-via-the-group-management "Remove users from a group via the group management").
+
+#### Prerequisites
+
+- You are logged in to a main account or a sandbox
+- You are granted to remover users from groups.
+
+#### Procedure
+
+*Settings > Users and groups > Tab BENUTZERVERWALTUNG (USER MANAGEMENT)*
+
+![User management](../../Assets/Screenshots/Settings/UsersGroups/UserManagement/UserManagement.png "[User management]")
+
+1. Click the user you want to remove from a group.   
+  The *Edit user "username"* view is displayed. By default, the *Profile* tab is preselected. 
+
+2. Click the *Groups* tab.   
+  The groups the user is assigned are listed.
+
+   ![Groups](../../Assets/Screenshots/Core1Platform/AdministratingCore1/UserGroups.png "[Groups]")  
+
+3. Select the group from which you want to remove the user.   
+  The toolbar is displayed.
+
+   ![Select group](../../Assets/Screenshots/Core1Platform/AdministratingCore1/UserSelectGroup.png "[Select group]")
+
+4. Click the ![Delete](../../Assets/Icons/Trash03.png  "[Delete]") (Delete) button.   
+  The user is removed from the selected group.<!--Julian, gibt es da noch einen Bestätigungsscreen?-->
+
+
+
+## Invite user to sandbox
+
+It is necessary to invite a user to a sandbox because users cannot be created there.
+When inviting a user to a sandbox, it is necessary to assign the user to a user group. Only then the user is listed in this instance and can access the sandbox. <!---Julian, ist das richtig oder habe ich etwas falsch verstanden? Ist User denn schon in Sandbox, ober prüft system in main account auf Plausibility?-->
```

Comment:
"it is necessary" umformuleren, um Wiederholung zu vermeiden (s. Satz direkt oben)

Reply:
done